src/libNode/FinalBlockProcessing.cpp

```diff
@@ -1443,13 +1454,15 @@ bool Node::ProcessMBnForwardTransaction(
        m_mediator.m_lookup->GetSyncType() == SyncType::NEW_LOOKUP_SYNC) ||
       (LOOKUP_NODE_MODE && !ARCHIVAL_LOOKUP &&
```

Comment:
Maybe the if condition can be combined here since there are 2 using `LOOKUP_NODE_MODE && ARCHIVAL_LOOKUP`

Reply:
well, one is `ARCHIVAL_LOOKUP` and other is `!ARCHIVAL_LOOKUP`